Automattic/jetpack
# VideoPress: Enable the resumable uploader

This diff enables the Resumable Uploader for VideoPress.

A follow up PR will be done to clean up the temporary feature gating code in the future if we no longer need a way to switch back to the original uploader.

Fixes # 1141-gh-Automattic/greenhouse

#### Changes proposed in this Pull Request:

* Enables the resumable uploader for all /video block uploads when VideoPress is enabled

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


nope

#### Testing instructions:






* On a new Jetpack site with VideoPress enabled
* Create or edit a post
* Add a `/video` block
* Upload a video, you should see the resumable uploader progress bar
* Use the pause button to stop and restart the upload
* Once the video reaches 100% it should convert to the Video block
* Sanity Check: Disable VideoPress, then upload a video with the `/video` block. It should upload with the old/core uploader
